Beginner_Projects/Amazon product availability checker/amazon.py

```
@@ -1,34 +1,34 @@
-    check_amazon_availability(product_url)
+import requests
+from bs4 import BeautifulSoup
+
+
+def check_amazon_availability(product_url):
+    headers = {
+        "User-Agent": "Your User Agent Here"  # Replace with a valid user agent string
+    }
+
+    try:
+        response = requests.get(product_url, headers=headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)